watcher/src/attestation_evidence_collector.rs

```diff
@@ -143,7 +165,7 @@ fn attestation_evidence_from_node_url(
         .try_next(&mut csprng, unverified_evidence_event)
         .map_err(|err| format!("Failed decoding attestation evidence from {node_url}: {err}"))?;
 
-    Ok(attestation_evidence)
+    Ok(attestation_evidence.into())
```

Comment:
Once #3602 comes in this will be getting an `Evidence` kind from the `Initiator`